docs/docsource/reference/backbeat/crr_retry.rst

```diff
@@ -15,7 +15,7 @@ This GET request retrieves a listing of failed operations at a site. Use
 this operation to learn if any CRR operations have failed at the site,
 and to retrieve the entire listing.
 
-**Request:** GET /_/backbeat/api/_/crr/failed?sitename=<site>&marker=<next-marker>
+**Request:** GET/_/backbeat/api/crr/failed?sitename=<site>&marker=<next-marker>
```

Comment:
Should there be a space after the method `GET`? The other requests have one

Reply:
Good spot.